src/Directions/DirectionsResponse.php

```diff
@@ -44,4 +44,9 @@ public function getRoutes(): array
     {
         return $this->routes;
     }
+
+    public function successful(): bool
+    {
+        return $this->getStatus() === 'OK' || $this->getStatus() === 'ZERO_RESULTS';
+    }
```

Comment:
From what i can remember, it is the same for each apis ? 
I mean this can be moved to the the super class or is there another reason ?

Reply:
Already not the same for Timezone API ;)

I took values from the Java implementation.

Reply:
Hmm not sure if we are speaking about the same thing. 
Status OK and ZERO_RESULTS are also in the TimeZone API and from what I've seen yet : in every other google maps api.
See https://developers.google.com/maps/documentation/timezone/intro#Responses

Reply:
Ok I've double checked and the elevation api does not have the zero_result but does it really matter ?
The second part will be false so only the OK will matter.

Reply:
In Timezone API, ZERO_RESULTS is not a successful status.

I don't know for the elevation api but https://github.com/googlemaps/google-maps-services-java/blob/master/src/main/java/com/google/maps/DirectionsApi.java#L74
I can remove it if you want